scripts/hooks/label-removal-guard.sh

```
@@ -0,0 +1,171 @@
+
+# Empty command → allow (nothing to gate).
+if [ -z "$COMMAND" ]; then exit 0; fi
+
+# Cheap pre-check: if the command doesn't mention any prohibited label,
+# skip the tokenize work entirely.
+case "$COMMAND" in
+  *needs-external-review*|*needs-human-review*|*policy-violation*) ;;
+  *) exit 0 ;;
+esac
+case "$COMMAND" in
+  *gh*pr*edit*|*gh*-R*pr*edit*|*gh*--repo*pr*edit*) ;;
```

Shellcheck (0.11.0)

[warning] This pattern always overrides a later one on line 68.

(SC2221)